onyo/commands/edit.py

```diff
@@ -129,15 +135,18 @@ def edit(args, opdir: str) -> None:
             If user wants to discard changes, restore the asset's state
             """
             repo._git(['restore', str(asset)])
-            print(f"'{asset}' not updated.")
+            if not args.quiet:
+                print(f"'{asset}' not updated.")
 
     # commit changes
     staged = sorted(repo.files_staged)
     if staged:
-        print(repo._diff_changes())
-        if request_user_response("Save changes? No discards all changes. (y/n) "):
-            repo.commit('edit asset(s).', staged)
-            repo.commit(repo.generate_commit_message(message=args.message, cmd="edit"))
+        if not args.quiet:
+            print(repo._diff_changes())
+        if args.yes or request_user_response("Save changes? No discards all changes. (y/n) "):
```

Comment:
It is not capitalized, because there is no default option. If the user response is neither y/n, then it will ask again. If the user wants to not having to answer either, `--yes` should be used.

Not having the "no" option as default, but instead asking again, is choosen so as to prevent the user from editing (or creating new assets) by hand, changing some lines, and then accidently answering `m` instead of `n` and thereby loosing the "progress" made.

This behavior was choosen at the time when we removed the temporary files. We can add a default option again if we decide to, but I want to mention that this is behaving like it does with intention and the message reflects the behavior.

Reply:
Ahh yes. I was not thinking about this in the context of `edit`. I agree that the "right" thing here is unknowable, and thus there should be no default.